examples/tasks/psutil-statistics-graphite.yml

```diff
@@ -0,0 +1,25 @@
+---
+version: 1
+schedule:
+  type: simple
+  interval: 1s
+max-failures: 1
+workflow:
+  collect:
+    metrics:
+      "/intel/psutil/load/load1": {}
+      "/intel/psutil/load/load15": {}
+      "/intel/psutil/load/load5": {}
+      "/intel/psutil/vm/available": {}
+      "/intel/psutil/vm/free": {}
+      "/intel/psutil/vm/used": {}
+    process:
+    - plugin_name: statistics
```

Comment:
Adding config for the statistics processor will make this a full fledged example with collector, processor and publisher.